tests/test_commons_auth.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import logging
+import sys
+import unittest
+from contextlib import contextmanager
+
+import connexion.apis.abstract
+import os
+import requests
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss
+from dss.logging import configure_test_logging
+from dss.util import UrlBuilder
+from tests import get_auth_header
+from tests.infra.server import ThreadedLocalServer
+from dss.config import override_bucket_config, BucketConfig, Replica
+from tests.infra import DSSAssertMixin, DSSUploadMixin, get_env, testmode
+
+logger = logging.getLogger(__name__)
+
```

Comment:
I think it is worthwhile providing a documentation block describing the approach we are using to testing auth.
Specifically:
- An authentication token is added implicitly and by default to `assertResponse` and that all the (other/standard) DSS tests are run using that authentication token.
- This test verifies that all APIs (for which authentication was added for commons) fail with a 403 if an authentication token is not provided.

Reply:
Added.